hawc/apps/common/templates/common/crispy_layout_filter_field.html

```diff
@@ -40,6 +41,12 @@
       <button type="submit" name="save" class="btn btn-primary px-5" title="Apply filters"><i class="fa fa-search" aria-hidden="true"></i></button>
     </div>
   </div>
+  {% for field in appended_fields %}
```

Comment:
Update, two commits:

* 1e27d9c6 add back if statement, but add catch-all else (41 seconds ago) {Andy Shapiro}
* 34723354 put in standard loop instead of multiple passes (24 minutes ago) {Andy Shapiro}